app/javascript/components/get-weather-query.jsx

```diff
@@ -96,6 +100,11 @@ WeatherQuery.propTypes = {
   }).isRequired,
   queryName: PropTypes.string.isRequired,
   cityName: PropTypes.string.isRequired,
+  startTimer: PropTypes.func,
+};
+
+WeatherQuery.defaultProps = {
+  startTimer: () => {},
```

Comment:
IIRC, this noop is used for the weatherCorner. Could we explicitly pass a `startTimer={() => {}}` instead of relying on the default? It makes it seem more intentional imo.